### ISSUE TITLE
Collapse/Expand model browser on demand

### ISSUE THREAD

**Issue by shobeira** [gaphor/gaphor#2595]
### Is your feature request related to a problem?
No.

### Describe the solution you'd like
In order to work in full screen on a diagram, it would be nice to be able to hide the model browser panel and widen the diagram visability.

### Describe alternatives you've considered
It is nice to be able to toggle property editor.

**Comment by danyeaw** [comment]
Hi @shobeira, would this be with a manual button press, like with the property editor, or an option to autohide the Model Browser?

**Comment by amolenaar** [comment]
You can already resize the model browser vertically. I suppose you do not want to hide the whole left bar, since it also contains the tool box. You can hide pretty much all of the model browser already.

![image](https://github.com/gaphor/gaphor/assets/96249/6d9dd63b-d570-4747-ab87-626acc16017d)

**Comment by shobeira** [comment]
That's right. I am already able to resize the panel, but I wish to be able to have the toggle button on the vertical edge of the model browser panel so that I would have more space to look at the diagram, concentrate on the diagram without having unnecessary visual elements around, and in order to see the whole picture of the diagram.
